### DISCUSSION TITLE
Mobile Version

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#2539] [General]
So I see theres a few discussions about mobile, but how do I actually set it up to use outside of my home/network?

I would love to test the capabailities from somewhere else as I get bored easily.

**Comment by vladmandic** [comment]
other than firing up browser on your mobile device and connecting to your server, this is a veeeery vague question.

**Reply by githubUser01946** [reply]
I will edit and elaborate on it more

**Comment by midcoastal** [comment]
Oh, look into the `--share` option.  I believe that is what you are looking for.

**Reply by githubUser01946** [reply]
Probably, thanks 🥇

**Comment by githubUser01946** [comment]
> Oh, look into the `--share` option. I believe that is what you are looking for.
